vllm/model_executor/models/llama.py

```diff
@@ -54,18 +56,21 @@ def __init__(
         hidden_size: int,
         intermediate_size: int,
         hidden_act: str,
+        quant_config: WeightQuantizationConfig=None
```

Comment:
Maybe not fixed in this PR: I feel passing `quant_config` into every class of every model is too big an overhead. We should find some other ways to seamlessly support quantization for different models.

Reply:
I have created AutoAWQ that automatically saves a quant_config.json and loads it from huggingface.
https://github.com/casper-hansen/AutoAWQ